src/Worker.Extensions.DurableTask/DefaultDurableClientContext.cs

```diff
@@ -26,8 +26,15 @@ private DefaultDurableClientContext(DurableTaskClient client, DurableClientInput
         this.Client = client ?? throw new ArgumentNullException(nameof(client));
         this.inputData = inputData ?? throw new ArgumentNullException(nameof(inputData));
 
-        ArgumentNullException.ThrowIfNull(inputData.taskHubName, nameof(inputData.taskHubName));
-        ArgumentNullException.ThrowIfNull(inputData.requiredQueryStringParameters, nameof(inputData.requiredQueryStringParameters));
+        if (string.IsNullOrEmpty(inputData.taskHubName))
+        {
+            throw new ArgumentNullException(nameof(inputData.taskHubName));
+        }
+
+        if (string.IsNullOrEmpty(inputData.requiredQueryStringParameters))
+        {
+            throw new ArgumentNullException(nameof(inputData.requiredQueryStringParameters));
+        }
```

Comment:
Does this change any behavior, or is this mostly just a style change?

Reply:
Technically it's a behavior change. Before we'd only throw if it was null. Now we throw if it's null or empty.